action.js

```diff
@@ -67,20 +67,26 @@ const getBodyFromGitHubContext = (team_id, github) => {
     }
   ];
 
+  console.log(get(github, ["context","payload"]))
+  process.exit();
+
   // Defines the HTTP request body to be used by the event with the matching
   // name.
   const event_bodies = {
-    "Change Initiated" : {
-      stage: "Change",
-      status: "Initiated",
-      change_id: get(github, ["context","payload","pull_request","head","ref"]),
-      stage_ref: get(github, ["context","payload","pull_request","head","ref"]),
+    "deployment succeeded" : {
       team_id,
-      custom: github
+      event_name: "deployment",
+      event_action: "succeeded",
+      repo: '',
+      environment: '',
+      branch: '',
+      commit: '',
+      image: '',
+      meta: github
```

Comment:
fields are fine for the payload in `uranium`, be sure to fill in the data instead of leaving it as empty strings.

Reply:
Yes, this still needs to be updated.